e2e/__tests__/v8Coverage.test.ts

```diff
@@ -31,8 +32,9 @@ onNodeVersions('>=10', () => {
           .join('\n') +
         '\n',
     ).toEqual(`
```

Comment:
Can we use `toMatchSnapshot` here?

Reply:
It differs between bode versions, so snapshots don't really work unfortunately. Maybe inline snapshots would work?

Reply:
👍  Let me have a try tomorrow

Reply:
Ah no, I've actually fixed so we can use a snapshot here in https://github.com/facebook/jest/commit/2edd5c4724cfadb944a6a233a7ad8fb5a3e05996#diff-305d58c78530d151f7bf398f775acc55 by using the same description. I just pushed up using snapshot here now 🙂